src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -74,9 +74,11 @@
     protected QoSEndpoint qos;
 
     //unique id assigned to every consumer during its creation.
-    public String consumerInstanceId;
+    protected String consumerInstanceId;
```

Comment:
the consumer-id is something needed just for the HTTP bridge so I think that we can move this field to the `HttpSinkBridgeEndpoint` class.